hw/ip/i2c/doc/_index.md

```diff
@@ -107,8 +107,13 @@ In this state the output drivers `scl_tx_o` and `sda_tx_o` are controlled direct
 When {{< regref OVRD.SCLVAL >}} and {{< regref OVRD.SDAVAL >}} are set high, the virtual open drain configuration will leave the output resistively pulled high, and controllable by remote targets.
 In this state, with SCL or SDA asserted high, the register fields {{< regref VAL.SCL_RX >}} and {{< regref VAL.SDA_RX >}} can be used to receive inputs (including remote acknowledgments) from target devices.
 
-### Byte-Formatted Programming Mode 
+### Host and Target Modes
```

Comment:
Could we separate into two sections dedicated to Host and Target mode which might be more readable?